Add CHANGELOG for v0.0.2 (#19)
turbot/steampipe-plugin-mastodon: CHANGELOG.md

> ## v0.1.0 [2023-05-25]

Reviewer: ```suggestion
## v0.0.2 [2023-05-25]
```

_Bug fixes_

> - Fixed `followers` and `following` columns in `mastodon_toot_favourite` table to be of `bigint` datatype. ([#16](https://github.com/turbot/steampipe-plugin-mastodon/pull/16))

Reviewer: ```suggestion
- Fixed the `followers` and the `following` columns in `mastodon_toot_favourite` table to be of `INT` datatype instead of `JSON`. ([#16](https://github.com/turbot/steampipe-plugin-mastodon/pull/16))
```

> - Fixed example query in `mastodon_my_list` table doc. ([#17](https://github.com/turbot/steampipe-plugin-mastodon/pull/17))

Reviewer: ```suggestion
- Fixed the example queries in `mastodon_my_list` table doc. ([#17](https://github.com/turbot/steampipe-plugin-mastodon/pull/17))
```
